app/views/veterans/thanks.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="container">
+  <div class="row">
+    <div class="col m12" role="main">
+      <h3  class="page-header">Thanks For Signing Up</h3>
+      <p class="lead">You'll receive an e-mail and an invitation to our Slack channel shortly.</p>
+      <p><a class="btn btn-action" href="<%= @slack_url %>">Download Slack</a></p>
```

Comment:
What happens if we have an unknown OS? Will this link be displayed with an invalid URL?

If so can we make it conditional? Display only if the link is not nil?